packages/charts/examples/components/demo/advanced-usage/events/event-sampler/event-sampler.component.html

```diff
@@ -13,14 +13,13 @@
                                [seriesRenderState]="chartAssist.renderStatesIndex[legendSeries.id]?.state"
                                [isSelected]="!chartAssist.isSeriesHidden(legendSeries.id)"
                                (isSelectedChange)="chartAssist.toggleSeries(legendSeries.id, $event)">
-                <nui-rich-legend-tile [value]="chartAssist.getHighlightedValue(legendSeries, 'y')"
-                                      [backgroundColor]="chartAssist.palette.standardColors.get(legendSeries.id)"
+                <nui-basic-legend-tile [value]="chartAssist.getHighlightedValue(legendSeries, 'y')"
                                       [color]="chartAssist.palette.textColors.get(legendSeries.id)">
```

Comment:
basic legend tile doesn't have the value or color inputs, so these can be removed:

```suggestion
                <nui-basic-legend-tile>
```